media-sound/airsonic-bin/Manifest

```diff
@@ -0,0 +1,5 @@
+AUX airsonic.confd 378 BLAKE2B 77a5ee351f95b78acfed1166ea8d276e3bf26aa9cfd5e222e784db52aeeeb5f374ffaeede9ef390ae4818b909973e20fa1cfa29f3fd3c995d4efff49d679946e SHA512 4aab17065ba45c04aa28bfc6839a9f75e0af4a1b7eb6aea879e438aa994f0677837f4e57d241b072a7cea701fc2ba6bd6be3586611de2c77672b03b0abadb82f
```

Comment:
You don't need lines like this. Just DIST.

Reply:
I know, but the new GLEP-74 defacto is not used until `ebuild [...] manifest` generate this.

Reply:
You can just commit only the DIST lines with ```git add -p```.

Reply:
For what? In this case you don’t need to use so much perfectionism

Reply:
I'm not trying to be a perfectionist. This is a rule for inclusion in ::gentoo.

Please [enable](https://wiki.gentoo.org/wiki/Repository_format/metadata/layout.conf#thin-manifests) thin-manifests.

Reply:
now I understand, sorry
fixed